src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -279,4 +279,17 @@
     <Exec Command="$(DacpacToolCommand)" />
   </Target>
   
+  <Target Name="CopyDacpacs" BeforeTargets="GetCopyToOutputDirectoryItems">
```

Comment:
We should probably add some comments here explaining what this target is for for future reference.